### DISCUSSION TITLE
Proposal for dealing with definitions and the SWEET organizational structure

### DISCUSSION THREAD

**Discussion by rduerr** [ESIPFed/sweet#259] [Ideas]
SWEET already has a hierarchy and thousands of terms, many of which have no definition.  The GCW cryospheric work made it painfully obvious that even within a discipline there may be very divergent definitions for some terms; divergent enough that not all definitions should fall under the SWEET branch where the term is currently placed.  

In these cases, it may make sense to have the same label represented in more than 1 SWEET branch with different IRI's.  

For example, taking the case of the cryospheric term Thermokarst which has two very different meanings in two different communities:

1. a landform
2. a process

SWEET currently has:

```

###  http://sweetontology.net/realmLandGlacial/Thermokarst
sorealg:Thermokarst rdf:type owl:Class ;
                 rdfs:subClassOf sorealg:GlacialLandform ;
                 rdfs:label "thermokarst"@en .
```

so any and all definitions that treat Thermokarst as a landform clearly belong here.  However, any definition that is about thermokarst as a process should go elsewhere in SWEET, perhaps as: 

```
###  http://sweetontology.net/procPhysical/Thermokarst
soprocp:Thermokarst rdf:type owl:Class .
                 rdfs:subClassOf soprocp:PhysicalProcess ;
                 rdfs:label "thermokarst"@en .
```
 **NOTE:** the second class in this example does not yet exist.  Applicable definitions would go under each branch of SWEET.

Please let me know if this proposal makes sense.  

I also note that we would be using whatever definitional pattern from #125 for dealing with definitions to ontology terms and their links (see for example this chunk of code from that issue):

```
               skos:definition  [ 
                   dcterms:description "A collection of data, published or curated by a single agent, and available for access or download in one or more representations"@en ;
                   dcterms:title "Dataset"@en ;
                   dcterms:source <https://www.w3.org/TR/vocab-dcat/#Class:Dataset> ;
                   dcterms:created "2021-08-25"^^xsd:date ;
             ]  ;

```

**Comment by semantic-nomad** [comment]
Yes, this makes sense to me.  Also note, particularly in this example, that explicit links to ENVO would be part of the pattern.

**Reply by rduerr** [reply]
as well as all of the links to the GCW glossaries that have term thermokarst in them.
